index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+
+</head>
+<body>
+  
+
+    <header class="nav">
+        <a href="" class="m-logo" >My Logo</a>
+    
+         <nav>
+           <ul>
+             <li class="link-right">
+                <a href="/someurl" class="inline"><b><img src="images/men.PNG" alt="ss"></b></a>
+             </li>
+        
+           </ul>
+        </nav>
+    </header>
+
+    <section class="section1">
+          <img src="images/l1.PNG" alt="abc" class="l1">
+          <img src="images/m.PNG" alt="abc" class="m">
+          <img src="images/o.PNG" alt="abc" class="m kap">
+
+          <img src="images/r6.png"  alt="abc" class="r6" >
+          <img src="images/e3.png"  alt="abc" class="image3 dkl" >
+    </section>
+
+
+     <section class="section2">
+          <h1 class="h2 imelex">I’m Aziz </h1>
+          <h2 class="h2 imelex gl">Glad to see you!</h2>
+          <p  class="sec3p1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+          <h5 class="ets" >ETS CONNECT</h5>
+          <img src="images/e12.PNG" alt="dd" class="bimg">
+        <ul class="social-menue">
+          <li><img src="images/twitter.PNG" alt="abc"></li>
+          <li><img src="images/linkedinicon.PNG" alt="abc"></li>
+          <li> <img src="images/medium.PNG" alt="abc"></li>
+          <li> <img src="images/github.PNG" alt="abc"></li>
+          <li><img src="images/angellist.PNG" alt="abc"></li>
```

Comment:
- [ ] Please fix the images because they are not loading correctly. Your image extension is capitalized that doesn't match the image file name. You can try `images/twitter.png` instead of `images/twitter.PNG`. Kindly fix all other image loading issues. :+1: